app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,44 @@
+class FavoritesController < ApplicationController
+
+  def index
+    @favorites = Favorite.all
```

Comment:
Does this action ever get called. Is there a page that shows all the favorites in the database?